feg/gateway/helm/feg-virtlet/README.md

```diff
@@ -162,7 +162,7 @@ The following table list the configurable parameters of the orchestrator chart a
     
    b. SSH into VM
    
-         ssh testuser@${feg_ip} "/var/opt/magma/docker/docker-compose exec magmad /usr/local/bin/show_gateway_info.py"
+         ssh testuser@${feg_ip} "cd /var/opt/magma/docker/ ; sudo docker compose exec magmad /usr/local/bin/show_gateway_info.py"
```

Comment:
Why is sudo now needed here?

Reply:
Was the code before not working and missing the sudo or is this somehow needed with the upgrade?